tools/webassembly.py

```diff
@@ -79,6 +79,17 @@ def helper(self, *args, **kwargs):
   return helper
 
 
+def type_to_sig(type):
+  # These should match the conversion in $sigToWasmTypes.
+  return {
+    Type.I32: 'i',
+    Type.I64: 'j',
+    Type.F32: 'f',
+    Type.F64: 'd',
+    Type.VOID: 'v'
+  }[type]
```

Comment:
Maybe put this in extract_metadata.py or emscripten.py?   webassembly.py is supposed to be just core wasm stuff.